### DISCUSSION TITLE
Dark mode

### DISCUSSION THREAD

**Discussion by kasper9n** [npm/feedback#764] [Website] [answered]
It's been asked and denied a few times already, but would be really nice to finally get a dark mode on npmjs.com

**Comment by ichenglin** [comment, marked as answer]
The **dark mode feature** has already been **suggested several times** (as u said),
and all of them are closed instantly by one of the maintainers with a similar response.

To me, it's clear that they have **something planned** with a **greater priority**,
therefore making a new suggestion **wouldn't really help** with the situation,
and it's likely that they would respond to you for the same reason as others.

> **[https://github.com/npm/feedback/discussions/656#discussioncomment-2453775](https://github.com/npm/feedback/discussions/656#discussioncomment-2453775)**
> at this time dark mode is not something we'll be prioritizing for npm. Thank you for the suggestion though 👍🏼

> **[https://github.com/npm/feedback/discussions/420#discussioncomment-1063808](https://github.com/npm/feedback/discussions/420#discussioncomment-1063808)**
> thanks for the suggestion. At this time we are not going to be able to prioritize Dark Mode on npmjs.com

> **[https://github.com/npm/feedback/discussions/231#discussioncomment-379016](https://github.com/npm/feedback/discussions/231#discussioncomment-379016)**
> Thanks for the suggestion. At this time the npm team will no be able to prioritize dark mode. While it is definitely a great feature we have a **[pretty full roadmap](https://github.com/npm/roadmap/projects/1)** and are focusing on smaller incremental improvements.

I'm **also looking forward** to having the **dark mode feature added**,
but maybe we should **respect their decision** on the priorities.

**Reply by kasper9n** [reply]
For sure, I posted a new one because I was thinking they might be able to have time for it now. Didn't see that the most recent was 5 months ago, though 5 months is still a lot of time

Hopefully what they've been focused on over the past 2 years is about to be done

**Reply by MylesBorins** [reply]
This hasn't change from a priority perspective. We are not currently prioritizing dark mode, nor is it likely to be prioritized in the near term.

**Reply by kasper9n** [reply]
Alright, that's unfortunate :( Hopefully soon

**Reply by dsshard** [reply]
Has the near future already arrived? A dark theme in 24 is very necessary)
